<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>964</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Галерея</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::CameraPhoto"/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="7">
         <widget class="QToolButton" name="P1_nextImage">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Наступне зображення</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>&gt;</string>
          </property>
         </widget>
        </item>
        <item row="2" column="5">
         <widget class="QToolButton" name="P1_sizeIcon">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Розмір на диску</string>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::DocumentSave"/>
          </property>
         </widget>
        </item>
        <item row="0" column="7">
         <widget class="QToolButton" name="P1_delete">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Видалити</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::EditDelete"/>
          </property>
         </widget>
        </item>
        <item row="2" column="7">
         <widget class="QToolButton" name="P1_resolutionIcon">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Розширення</string>
          </property>
          <property name="styleSheet">
           <string notr="true">border:none</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::ZoomFitBest"/>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QToolButton" name="P1_zoomOut">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Зменшити масштаб</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::ZoomOut"/>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QToolButton" name="P1_zoomIn">
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Збільшити масштаб</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset theme="QIcon::ThemeIcon::ZoomIn"/>
          </property>
         </widget>
        </item>
        <item row="2" column="4">
         <widget class="QLabel" name="P1_size">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="6">
         <widget class="QLabel" name="P1_resolution">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignRight|Qt::AlignmentFlag::AlignTrailing|Qt::AlignmentFlag::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="1" colspan="6">
         <widget class="QLabel" name="P1_mainImageView">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>600</height>
           </size>
          </property>
          <property name="text">
           <string>Зображення не завантажено</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QToolButton" name="P1_prevImage">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="focusPolicy">
           <enum>Qt::FocusPolicy::NoFocus</enum>
          </property>
          <property name="toolTip">
           <string>Попереднє зображення</string>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>&lt;</string>
          </property>
         </widget>
        </item>
        <item row="0" column="2" colspan="5">
         <widget class="QLabel" name="P1_name">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="P1_scale">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Масштаб: 100%</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QToolButton" name="P1_toList">
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Відкрити список</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::FormatJustifyCenter"/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="P1_rotate">
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="toolTip">
             <string>Повернути зображення</string>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="QIcon::ThemeIcon::ViewRefresh"/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="2">
         <widget class="QGroupBox" name="searchGroupBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string/>
          </property>
          <layout class="QFormLayout" name="formLayout">
           <item row="0" column="0" colspan="2">
            <widget class="QLineEdit" name="P2_widthFilter">
             <property name="placeholderText">
              <string>Ширина</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="P2_heightFilter">
             <property name="placeholderText">
              <string>Висота</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QLineEdit" name="P2_sizeFilter">
             <property name="placeholderText">
              <string>Розмір (KB)</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0" colspan="2">
            <widget class="QComboBox" name="P2_formatFilter">
             <item>
              <property name="text">
               <string>Формат</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>BMP</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>PNG</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>JPEG</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>JPG</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="4" column="0" colspan="2">
            <widget class="QPushButton" name="P2_search">
             <property name="text">
              <string>Пошук</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QListWidget" name="P2_imageListView">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="frameShape">
           <enum>QFrame::Shape::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Shadow::Plain</enum>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustIgnored</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::EditTrigger::DoubleClicked|QAbstractItemView::EditTrigger::EditKeyPressed|QAbstractItemView::EditTrigger::SelectedClicked</set>
          </property>
          <property name="showDropIndicator" stdset="0">
           <bool>false</bool>
          </property>
          <property name="dragDropMode">
           <enum>QAbstractItemView::DragDropMode::DragDrop</enum>
          </property>
          <property name="defaultDropAction">
           <enum>Qt::DropAction::IgnoreAction</enum>
          </property>
          <property name="alternatingRowColors">
           <bool>false</bool>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
          </property>
          <property name="iconSize">
           <size>
            <width>64</width>
            <height>64</height>
           </size>
          </property>
          <property name="verticalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="horizontalScrollMode">
           <enum>QAbstractItemView::ScrollMode::ScrollPerPixel</enum>
          </property>
          <property name="movement">
           <enum>QListView::Movement::Free</enum>
          </property>
          <property name="flow">
           <enum>QListView::Flow::LeftToRight</enum>
          </property>
          <property name="resizeMode">
           <enum>QListView::ResizeMode::Adjust</enum>
          </property>
          <property name="layoutMode">
           <enum>QListView::LayoutMode::SinglePass</enum>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="gridSize">
           <size>
            <width>100</width>
            <height>100</height>
           </size>
          </property>
          <property name="viewMode">
           <enum>QListView::ViewMode::IconMode</enum>
          </property>
          <property name="modelColumn">
           <number>0</number>
          </property>
          <property name="sortingEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="3" rowspan="3">
         <widget class="QGroupBox" name="groupBox">
          <property name="minimumSize">
           <size>
            <width>300</width>
            <height>256</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>256</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="title">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="2" column="0">
            <widget class="QLabel" name="P2_detailsTitle">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:700;&quot;&gt;Деталі:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="P2_imageDetails">
             <property name="minimumSize">
              <size>
               <width>256</width>
               <height>0</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::NoFrame</enum>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QRadioButton" name="P2_inAscending">
             <property name="text">
              <string>За зростанням</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="P2_imagePreview">
             <property name="minimumSize">
              <size>
               <width>256</width>
               <height>256</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>256</width>
               <height>256</height>
              </size>
             </property>
             <property name="frameShape">
              <enum>QFrame::Shape::NoFrame</enum>
             </property>
             <property name="midLineWidth">
              <number>0</number>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QRadioButton" name="P2_inDescending">
             <property name="text">
              <string>За спаданням</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0" colspan="2">
            <widget class="QComboBox" name="P2_sort">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="currentText">
              <string>Сортувати за іменем</string>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <property name="frame">
              <bool>true</bool>
             </property>
             <item>
              <property name="text">
               <string>Сортувати за іменем</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Сортувати за розміром</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Сортувати за шириною</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Сортувати з висотою</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="6" column="0" colspan="2">
            <widget class="QLineEdit" name="P2_minSize">
             <property name="text">
              <string/>
             </property>
             <property name="placeholderText">
              <string>Мінімальний розмір (МБ)</string>
             </property>
            </widget>
           </item>
           <item row="7" column="0" colspan="2">
            <widget class="QLineEdit" name="P2_maxSize">
             <property name="placeholderText">
              <string>Максимальний розмір (МБ)</string>
             </property>
            </widget>
           </item>
           <item row="8" column="0" colspan="2">
            <widget class="QPushButton" name="P2_searchInSizeRange">
             <property name="text">
              <string>Зображення у заданому діапазоні</string>
             </property>
            </widget>
           </item>
           <item row="9" column="0" colspan="2">
            <widget class="QPushButton" name="P2_displayMinMaxSize">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="text">
              <string>Вивести зображення з найменшим
 і найбільшим розміром на диску</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="Line" name="P2_separator">
             <property name="orientation">
              <enum>Qt::Orientation::Horizontal</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>964</width>
     <height>25</height>
    </rect>
   </property>
   <property name="nativeMenuBar">
    <bool>false</bool>
   </property>
   <widget class="QMenu" name="File">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>115</y>
      <width>280</width>
      <height>200</height>
     </rect>
    </property>
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="M_openFolder"/>
    <addaction name="M_createObject"/>
    <addaction name="separator"/>
    <addaction name="M_exportToFile"/>
    <addaction name="M_importFromFile"/>
    <addaction name="separator"/>
    <addaction name="M_changeFormat"/>
   </widget>
   <addaction name="File"/>
  </widget>
  <action name="M_exportToFile">
   <property name="text">
    <string>Експорт даних у файл</string>
   </property>
  </action>
  <action name="M_importFromFile">
   <property name="text">
    <string>Імпорт даних із файлу</string>
   </property>
  </action>
  <action name="M_changeFormat">
   <property name="text">
    <string>Експорт файлу в інший формат</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Відкрити папку</string>
   </property>
  </action>
  <action name="M_openFolder">
   <property name="text">
    <string>Відкрити папку</string>
   </property>
  </action>
  <action name="M_createObject">
   <property name="text">
    <string>Створити об'єкт вручну</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
